glean_parser/schemas/metrics.1-0-0.schema.yaml

```diff
@@ -97,6 +97,8 @@ definitions:
 
             - `uuid`: Record a UUID v4.
 
+            - `jwe`: Record a JWE value.
```

Comment:
Nit: Maybe link to the JWE standard here?  (We can use markdown in this file...)